app/services/person_csv_importer.rb

```diff
@@ -6,7 +6,15 @@ class PersonCsvImporter
   extend Forwardable
 
   REQUIRED_COLUMNS = %i(given_name surname email).freeze
-  OPTIONAL_COLUMNS = %i(primary_phone_number pager_number building location_in_building city).freeze
+  OPTIONAL_COLUMNS = %i(
+    primary_phone_number
+    secondary_phone_number
+    pager_number building
```

Comment:
Could be split so that each line is a single field, it might get lost otherwise.